godot/addons/quentincaffeino-console/src/Console.gd

```diff
@@ -11,6 +11,14 @@ const IntRangeType = preload('Type/IntRangeType.gd')
 const FloatRangeType = preload('Type/FloatRangeType.gd')
 const FilterType = preload('Type/FilterType.gd')
 
+# Signals
+signal opened
+signal closed
+signal toggled
```

Comment:
Maybe it would be better to pass an argument here cause user would need to store state somewhere or ask console which would require him writing extra line, what do you think?

Reply:
When you say new state here do you mean pass in the console? Or some other variable?

Reply:
I mean [this](https://github.com/quentincaffeino/godot-console/pull/38/files#diff-2cd4cefa1a0e38ed09c374122b899ca03535e971a163cee9e19affa083db634bR218),
so it would look something like this:
```gdscript
	signal toggled(is_console_shown)

...

	is_console_shown = !self.is_console_shown
	emit_signal("toggled", is_console_shown)
```

what do you think?

Reply:
Ah! I see the benefits of this. Making it only a toggle and passing is_active seems like a good idea. I can add those changes.